docs/reference/server-configuration.md

```diff
@@ -525,6 +525,17 @@ becomes `_confluent-ksql-default__command_topic`).
     separator character of some form, like a dash or underscore, as
     this makes the internal topic names easier to read.
 
+## `ksql.service.name`
+
+**Per query:** yes
+
+The service NAME of the ksqlDB server. This is used for RBAC authorization
+instead of the ksqlDB service ID above. This requires each ksqlDB cluster
+to have a mandatory and unique name within the same logical {{ site.ak }} 
+cluster.
+
+By default, the service NAME of KsqlDB servers is `default_service_name`.
```

Comment:
```suggestion
By default, the service NAME of ksqlDB servers is `default_service_name`.
```

Reply:
Let's remove the comment about used for RBAC authorization ...

The ID vs Name are used for authorization based on the external authorization plugin. It's up to the implementator to decide what to do. In Ksql without running in Cloud, this config does not do anything, so let's just keep a generic comment.